functionary/ui/templates/core/scheduled_task_create.html

```diff
@@ -0,0 +1,220 @@
+{% extends "base.html" %}
+{% load static %}
+{% load unicorn %}
+{% block content %}
+<div class="block">
+    {% if form %}
+        <form id="scheduled-task-create-form"
+                method="post"
+                action="{% url 'ui:create-scheduled-task' %}">
+            {% csrf_token %}
+            
+            <!-- Non-Field Errors display -->
+            <div class="columns is-centered">
+                <div class="column has-text-centered is-half">
+                    {% if form.non_field_errors %}
+                    <div class="notification is-warning has-text-black">
+                        <button class="delete" onclick="return this.parentNode.remove();"></button>
+                        Non-Field Errors:
+                        {{ form.non_field_errors }}
+                    </div>
+                    {% endif %}
+                </div>
+            </div>
+
+            <div class="box">
+
+                <!-- Name row/field -->
+                <div class="field">
+                    <div class="columns is-8">
+                        <div class="column is-one-third">
+                            <label for="{{ form.name.id_for_label }}">
+                                <b {% if form.name.errors %}class="errors"{% endif %} class="title">{{ form.name.label }}</b> 
```

Comment:
The sizing of the title and subtitle end up being quite large and inconsistent with the other pages throughout the UI.  Can we shrink them a bit to make it feel more cohesive?  The same might be true of the text in the input boxes.

Reply:
I left the form labels as title/subtitles, but modified their sizes. Let me know if you think they look better. 

I chose to leave the input boxes the same, since I think the `is-small` modifier is way too small